device/mellanox/x86_64-mlnx_msn2700a1-r0/sensors.conf

```diff
@@ -21,6 +21,42 @@ bus "i2c-17" "i2c-1-mux (chan_id 8)"
 # Temperature sensors
 chip "mlxsw-i2c-*-48"
     label temp1 "Ambient ASIC Temp"
+    ignore temp2
```

Comment:
There are 32 ports in 2700a1, but there are 35 ignore here

Reply:
changed